### ISSUE TITLE
cssa is not a function

### ISSUE THREAD

**Issue by craigcosmo** [postcss/postcss-loader#169]
this loader was good untill I put this this style in

```
.shadow{
	-webkit-filter: drop-shadow( 3px 3px 2px rgba(0,0,0,.7) );
	filter: drop-shadow( 3px 3px 2px rgba(0,0,0,.7) ); 
}
```
error 

<img width="566" alt="screen shot 2017-02-07 at 10 17 16 pm" src="https://cloud.githubusercontent.com/assets/538657/22697161/3888caa2-ed83-11e6-828c-8520c056d89e.png">

webpack config
```
{
	test: /\.(css)$/,
	loader: ExtractTextPlugin.extract({
		fallbackLoader: 'style-loader',
		loader: [
			{ loader: 'css-loader?sourceMap&importLoaders=1'},
			{ loader: 'postcss-loader?sourceMap&parser=postcss-scss'},
		]
	})
},
```

error text:

```
Child extract-text-webpack-plugin:
    chunk    {0} extract-text-webpack-plugin-output-filename 1.38 kB [entry]
        [0] ./~/css-loader?sourceMap&importLoaders=1!./~/postcss-loader?sourceMap&parser=postcss-scss!./app/style/page.css 1.38 kB {0} [built] [failed] [1 error]
    
    ERROR in ./~/css-loader?sourceMap&importLoaders=1!./~/postcss-loader?sourceMap&parser=postcss-scss!./app/style/page.css
    Module build failed: TypeError: oneColor(...).cssa is not a function
        at Object.Filter.filters.dropShadow (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:401:36)
        at Filter.convert (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:543:30)
        at /Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:575:36
        at /lib/container.es6:152:28
        at /lib/container.es6:110:26
        at Rule.each (/lib/container.es6:78:22)
        at Rule.walk (/lib/container.es6:109:21)
        at Rule.walkDecls (/lib/container.es6:150:25)
        at /Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:556:8
        at /lib/container.es6:199:28
        at /lib/container.es6:110:26
        at Root.each (/lib/container.es6:78:22)
        at Root.walk (/lib/container.es6:109:21)
        at Root.walkRules (/lib/container.es6:197:25)
        at Filter.postcss (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:554:6)
        at LazyResult.run (/lib/lazy-result.es6:321:20)
        at /lib/lazy-result.es6:248:32
        at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)
        at /lib/lazy-result.es6:260:22
        at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)
        at /lib/lazy-result.es6:260:22
        at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)

```

**Comment by ai** [comment]
@craigcosmo post a full stacktrack please

**Comment by craigcosmo** [comment]
it's the space
**error**
```
-webkit-filter: drop-shadow( 3px 3px 2px rgba(0,0,0,.7) );
	filter: drop-shadow( 3px 3px 2px rgba(0,0,0,.7) ); 
```

**no error**
```
-webkit-filter: drop-shadow( 3px 3px 2px rgba(0,0,0,.7));
	filter: drop-shadow( 3px 3px 2px rgba(0,0,0,.7)); 
```

**Comment by ai** [comment]
Show full console output when you get this error.

**Comment by craigcosmo** [comment]
```
Hash: d17eaf90a01068ed5753                                                    e Version: webpack 2.2.1
Time: 1818ms
         Asset       Size  Chunks                    Chunk Names
     bundle.js     2.5 MB       0  [emitted]  [big]  main
    bundle.css    20.9 kB       0  [emitted]         main
 bundle.js.map    3.03 MB       0  [emitted]         main
bundle.css.map    39.8 kB       0  [emitted]         main
    index.html  478 bytes          [emitted]         
chunk    {0} bundle.js, bundle.css, bundle.js.map, bundle.css.map (main) 2.42 MB [entry] [rendered]
  [279] ./app/style/page.css 1.94 kB {0} [built] [failed] [1 error]
     + 777 hidden modules

ERROR in ./app/style/page.css
Module build failed: ModuleBuildError: Module build failed: TypeError: oneColor(...).cssa is not a function
    at Object.Filter.filters.dropShadow (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:401:36)
    at Filter.convert (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:543:30)
    at /Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:575:36
    at /lib/container.es6:152:28
    at /lib/container.es6:110:26
    at Rule.each (/lib/container.es6:78:22)
    at Rule.walk (/lib/container.es6:109:21)
    at Rule.walkDecls (/lib/container.es6:150:25)
    at /Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:556:8
    at /lib/container.es6:199:28
    at /lib/container.es6:110:26
    at Root.each (/lib/container.es6:78:22)
    at Root.walk (/lib/container.es6:109:21)
    at Root.walkRules (/lib/container.es6:197:25)
    at Filter.postcss (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:554:6)
    at LazyResult.run (/lib/lazy-result.es6:321:20)
    at /lib/lazy-result.es6:248:32
    at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)
    at /lib/lazy-result.es6:260:22
    at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)
    at /lib/lazy-result.es6:260:22
    at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)
    at /Users/craigcosmo/Desktop/lark chat app/node_modules/webpack/lib/NormalModule.js:141:35
    at /Users/craigcosmo/Desktop/lark chat app/node_modules/loader-runner/lib/LoaderRunner.js:364:11
    at /Users/craigcosmo/Desktop/lark chat app/node_modules/loader-runner/lib/LoaderRunner.js:230:18
    at context.callback (/Users/craigcosmo/Desktop/lark chat app/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at /Users/craigcosmo/Desktop/lark chat app/node_modules/postcss-loader/index.js:141:13
 @ ./app/component/Page.js 20:0-28
 @ ./app/redux/MainContainer.js
 @ ./app/index.js
 @ multi (webpack)-dev-server/client?http://localhost:1114 ./app/index.js

ERROR in ./app/style/page.css
Module build failed: ModuleBuildError: Module build failed: TypeError: oneColor(...).cssa is not a function
    at Object.Filter.filters.dropShadow (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:401:36)
    at Filter.convert (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:543:30)
    at /Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:575:36
    at /lib/container.es6:152:28
    at /lib/container.es6:110:26
    at Rule.each (/lib/container.es6:78:22)
    at Rule.walk (/lib/container.es6:109:21)
    at Rule.walkDecls (/lib/container.es6:150:25)
    at /Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:556:8
    at /lib/container.es6:199:28
    at /lib/container.es6:110:26
    at Root.each (/lib/container.es6:78:22)
    at Root.walk (/lib/container.es6:109:21)
    at Root.walkRules (/lib/container.es6:197:25)
    at Filter.postcss (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:554:6)
    at LazyResult.run (/lib/lazy-result.es6:321:20)
    at /lib/lazy-result.es6:248:32
    at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)
    at /lib/lazy-result.es6:260:22
    at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)
    at /lib/lazy-result.es6:260:22
    at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)
    at /Users/craigcosmo/Desktop/lark chat app/node_modules/webpack/lib/NormalModule.js:141:35
    at /Users/craigcosmo/Desktop/lark chat app/node_modules/loader-runner/lib/LoaderRunner.js:364:11
    at /Users/craigcosmo/Desktop/lark chat app/node_modules/loader-runner/lib/LoaderRunner.js:230:18
    at context.callback (/Users/craigcosmo/Desktop/lark chat app/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at /Users/craigcosmo/Desktop/lark chat app/node_modules/postcss-loader/index.js:141:13

ERROR in /Users/craigcosmo/Desktop/lark chat app/node_modules/extract-text-webpack-plugin/loader.js?{"omit":1,"remove":true}!/Users/craigcosmo/Desktop/lark chat app/node_modules/style-loader/index.js!/Users/craigcosmo/Desktop/lark chat app/node_modules/css-loader/index.js?sourceMap&importLoaders=1!/Users/craigcosmo/Desktop/lark chat app/node_modules/postcss-loader/index.js?sourceMap&parser=postcss-scss!/Users/craigcosmo/Desktop/lark chat app/app/style/page.css doesn't export content
Child html-webpack-plugin for "index.html":
    chunk    {0} index.html 541 kB [entry]
         + 4 hidden modules
Child extract-text-webpack-plugin:
    chunk    {0} extract-text-webpack-plugin-output-filename 1.38 kB [entry] [rendered]
        [0] ./~/css-loader?sourceMap&importLoaders=1!./~/postcss-loader?sourceMap&parser=postcss-scss!./app/style/page.css 1.38 kB {0} [built] [failed] [1 error]
    
    ERROR in ./~/css-loader?sourceMap&importLoaders=1!./~/postcss-loader?sourceMap&parser=postcss-scss!./app/style/page.css
    Module build failed: TypeError: oneColor(...).cssa is not a function
        at Object.Filter.filters.dropShadow (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:401:36)
        at Filter.convert (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:543:30)
        at /Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:575:36
        at /lib/container.es6:152:28
        at /lib/container.es6:110:26
        at Rule.each (/lib/container.es6:78:22)
        at Rule.walk (/lib/container.es6:109:21)
        at Rule.walkDecls (/lib/container.es6:150:25)
        at /Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:556:8
        at /lib/container.es6:199:28
        at /lib/container.es6:110:26
        at Root.each (/lib/container.es6:78:22)
        at Root.walk (/lib/container.es6:109:21)
        at Root.walkRules (/lib/container.es6:197:25)
        at Filter.postcss (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:554:6)
        at LazyResult.run (/lib/lazy-result.es6:321:20)
        at /lib/lazy-result.es6:248:32
        at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)
        at /lib/lazy-result.es6:260:22
        at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)
        at /lib/lazy-result.es6:260:22
        at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)
Child extract-text-webpack-plugin:
    chunk    {0} extract-text-webpack-plugin-output-filename 1.38 kB [entry]
        [0] ./~/css-loader?sourceMap&importLoaders=1!./~/postcss-loader?sourceMap&parser=postcss-scss!./app/style/page.css 1.38 kB {0} [built] [failed] [1 error]
    
    ERROR in ./~/css-loader?sourceMap&importLoaders=1!./~/postcss-loader?sourceMap&parser=postcss-scss!./app/style/page.css
    Module build failed: TypeError: oneColor(...).cssa is not a function
        at Object.Filter.filters.dropShadow (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:401:36)
        at Filter.convert (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:543:30)
        at /Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:575:36
        at /lib/container.es6:152:28
        at /lib/container.es6:110:26
        at Rule.each (/lib/container.es6:78:22)
        at Rule.walk (/lib/container.es6:109:21)
        at Rule.walkDecls (/lib/container.es6:150:25)
        at /Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:556:8
        at /lib/container.es6:199:28
        at /lib/container.es6:110:26
        at Root.each (/lib/container.es6:78:22)
        at Root.walk (/lib/container.es6:109:21)
        at Root.walkRules (/lib/container.es6:197:25)
        at Filter.postcss (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:554:6)
        at LazyResult.run (/lib/lazy-result.es6:321:20)
        at /lib/lazy-result.es6:248:32
        at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)
        at /lib/lazy-result.es6:260:22
        at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)
        at /lib/lazy-result.es6:260:22
        at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)

```

**Comment by ai** [comment]
@iamvdo seems like this issue from `pleeease-filters`

**Comment by ai** [comment]
@craigcosmo it is stacktrace:

```
    ERROR in ./~/css-loader?sourceMap&importLoaders=1!./~/postcss-loader?sourceMap&parser=postcss-scss!./app/style/page.css
    Module build failed: TypeError: oneColor(...).cssa is not a function
        at Object.Filter.filters.dropShadow (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:401:36)
        at Filter.convert (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:543:30)
        at /Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:575:36
        at /lib/container.es6:152:28
        at /lib/container.es6:110:26
        at Rule.each (/lib/container.es6:78:22)
        at Rule.walk (/lib/container.es6:109:21)
        at Rule.walkDecls (/lib/container.es6:150:25)
        at /Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:556:8
        at /lib/container.es6:199:28
        at /lib/container.es6:110:26
        at Root.each (/lib/container.es6:78:22)
        at Root.walk (/lib/container.es6:109:21)
        at Root.walkRules (/lib/container.es6:197:25)
        at Filter.postcss (/Users/craigcosmo/Desktop/lark chat app/node_modules/pleeease-filters/index.js:554:6)
        at LazyResult.run (/lib/lazy-result.es6:321:20)
        at /lib/lazy-result.es6:248:32
        at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)
        at /lib/lazy-result.es6:260:22
        at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)
        at /lib/lazy-result.es6:260:22
        at LazyResult.asyncTick (/lib/lazy-result.es6:246:13)
```

Always post it in issues.

**Comment by ai** [comment]
@craigcosmo this issue is not came from `postcss-loader`. It came from PostCSS plugin, that you use inside loader. Please open issue here: https://github.com/iamvdo/pleeease-filters

**Comment by iamvdo** [comment]
Fixed in 3.0.1 :+1:
